index.html

```diff
@@ -868,7 +952,7 @@ <h3>Controllers</h3>
 While the identifier used for a [=controller=] is unambiguous, this does not
```

Comment:
```suggestion
While the identifier used for a [=documentController=] is unambiguous, this does not
```

Reply:
-1 for the reasons stated here: https://github.com/w3c/controller-document/pull/116/files#r1839014775